lib/puppet-lint/plugins/check_whitespace/multiple_whitespace.rb

```diff
@@ -0,0 +1,27 @@
+# Public: Check the manifest tokens for multiple spaces in whitespace that is not after a newline and record a warning for each instance found.
+# #
+# https://docs.puppet.com/guides/style_guide.html#spacing-indentation-and-whitespace
+PuppetLint.new_check(:multiple_whitespace) do
```

Comment:
I'll see if I can make it work, and I agree, there are probably at least that and more exceptions. If I can't get a handle on it I'll restrict it to just whitespace around titles and resources, but I was hoping to make the check more general if possible.